services/recordandreplay.md

```diff
@@ -0,0 +1,44 @@
+# Record and Replay
+
+    identifier: 0x1b72bf50
+    group: sound
+    tags: 8bit, padauk
+    
+A record and replay module. You can record a few seconds of audio and play it back.
+
+## Commands
+
+    command replay @ 0x80 {}
+    
+Replay cached audio.
+    
+    command record @ 0x81 {
+        milliseconds: u16 ms
+    }
+    
+Record audio for N milliseconds.
+
+    cancel record @ 0x82 {}
```

Comment:
```
command clear @ 0x82 {}
```
It clears the recorded soud right?

Reply:
No, just interrupt the current record session and keep the recorded samples. Our current implement is just a die chip with only 2 control io. But I think some advanced implement should be able to read/write/delete and list sound samples, stored in onboard spi-flash or SD card. Maybe I should check similar solutions in the market to make this service more complete~

Reply:
oh great, let's do
```
command stop_recording @ 0x82 {}
```